packages/create-hint/src/templates/meta.ts.hbs

```diff
@@ -1,21 +1,43 @@
 import { Category } from 'hint/dist/src/lib/enums/category';
 import { HintScope } from 'hint/dist/src/lib/enums/hint-scope';
 import { HintMetadata } from 'hint/dist/src/lib/types';
+{{#if packageData.official}}
+
+import { getMessage } from './i18n.import';{{/if}}
 
 const meta: HintMetadata = {
     docs: {
-        category: Category.{{category}},
-        description: `{{description}}`
+        category: Category.{{hint.category}},{{#if packageData.official}}
+        description: getMessage('{{#if packageData.isMulti}}{{hint.normalizedName}}_{{/if}}description', 'en'),
```

Comment:
Same as before, it is not `normalizedName`. There are more places where you need to change it.